src/js/component.js

```diff
@@ -1398,14 +1399,26 @@ class Component {
    * @method getComponent
    */
   static getComponent(name) {
-    if (Component.components_ && Component.components_[name]) {
-      return Component.components_[name];
+    if (!name) {
```

Comment:
i think something was trying to retrieve a component by name `''`, but i couldn't track it in the call stack.

might need refactoring, but i got out early and updated the implementation of `toTitleCase`.